packages/theme/src/components/modal.ts

```diff
@@ -1,19 +1,19 @@
 import clsx from 'clsx'
 
 const modalOverlayClsx = ({ className = '' }: { className?: string } = {}) =>
-  clsx('fixed inset-0 z-50 bg-black/40', className)
+  clsx('fixed inset-0 z-50 bg-background-backdrop/40', className)
```

Comment:
```suggestion
  clsx('fixed inset-0 z-50 bg-background-backdrop', className)
```

The `backdrop` color already has an alpha channel